tests/demo.py

```diff
@@ -258,8 +268,179 @@ def test_dead_old_tail_when_adding():
     proc.kill()
 
 
-def BenchCredis(num_nodes, num_ops, num_clients):
-    common.Start(chain=common.MakeChain(num_nodes))
+def test_etcd_kill_middle(startcredis_etcdonly):
```

Comment:
Hmm, do these etcd unit tests need the caller to manually start etcd outside?  Seems like so, since when I run `python2 -m pytest demo.py` there are 4 passes, 9 errors. 

How hard is it to have common to take care of starting etcd (activated behind env var, see other comments)?

Reply:
It's not impossible, but it means that you need to have etcd installed or we need to package it as a binary with credis. I'm leaning toward just packaging the Goless version from the readme and adding a setup script.